testng/README.md

```diff
@@ -48,4 +47,19 @@ Cucumber provides limited support for [SkipException](https://jitpack.io/com/git
 
 * Throwing a `SkipException` results in both Cucumber and TestNG marking the test as skipped.
 * Throwing a subclass of `SkipException` results in Cucumber marking the test as failed and TestNG marking the test 
-as skipped.
\ No newline at end of file
+as skipped.
+
+## Parallel execution ##
+
+Cucumber TestNG supports parallel execution of scenarios. Override the `scenarios` method to enable parallel execution.
+
+```java
+public class RunCukesTest extends AbstractTestNGCucumberTests {
+
+    @Override
+    @DataProvider(parallel = true)
```

Comment:
@kathyrollo you've been using TestNG more then I have. Is this the only way to make tests  with a a data provider run in parallel? Do you happen to know if we can control this from maven instead?